phpunit.xml.dist

```diff
@@ -16,6 +16,7 @@
 	<testsuite name="common">
 		<directory>tests/Common</directory>
 		<file>tests/File/CommonFileTests.php</file>
+		<file>tests/Backend/Workspaces/ReadOnlyUserTest.php</file>
```

Comment:
nemohlo to být parallelní?

Reply:
WorkspacesTestCase není ready pro paralelní zpracování + by musel upravit setup pro travis a nakonfigurovat token pro novyháho usera v paratest-common

Reply:
Aha, jasně.